hedera-node/hedera-config/src/main/java/com/hedera/node/config/sources/PropertyConfigSource.java

```diff
@@ -87,4 +88,21 @@ public String getValue(@NonNull String key) throws NoSuchElementException {
     public int getOrdinal() {
         return ordinal;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
```

Comment:
Does a unit test need to be made to get these warnings to go away?  Or suppress warning flag?  

<img width="1561" alt="Screenshot 2023-08-21 at 11 08 13 AM" src="https://github.com/hashgraph/hedera-services/assets/123979964/36a87b3d-07e5-44f4-9c9f-4fde3abd29e4">